packages/jest-cli/src/TestRunner.js

```diff
@@ -266,6 +265,7 @@ class TestRunner {
     }
 
     if (this._config.collectCoverage) {
+      const CoverageReporter = require('./reporters/CoverageReporter');
```

Comment:
if we're not in `collectCoverage` mode we don't need any of these. and dependency graph of istanbul is pretty big. that will optimize the start time